### PR TITLE
Re-enable aggregation on JARVIS

### PR DESCRIPTION
**Pull request by ml-evs**
Undoes #79 and enables query aggregation on JARVIS again. Need to verify first with @knc6 that filters and pagination are working again.

### CODE REVIEW

**Comment by merkys** [comment]
Looks legit, but why the OPTIMADE validator is failing?

File: src/index-metadbs/jarvis/v1/links.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
             "base_url": "https://jarvis.nist.gov/optimade/jarvisdft",
             "homepage": "https://jarvis.nist.gov",
             "link_type": "child",
```
Comment (ml-evs) [inline src/index-metadbs/jarvis/v1/links.json:19]:
```suggestion
            "link_type": "child"
```

**Review by merkys — APPROVED** [review]

**Comment by ml-evs** [comment]
I'll actually hold off merging here until @knc6 confirms it is ready

**Comment by knc6** [comment]
@ml-evs IMO its ready, please go ahead and merge.

**Pull request merged**